injector/network_disruption_test.go

```diff
@@ -354,6 +354,33 @@ var _ = Describe("Failure", func() {
 			})
 		})
 
+		Context("when resolved host IPs change", func() {
+			BeforeEach(func() {
+				spec.Hosts = []v1beta1.NetworkDisruptionHostSpec{
+					{
+						Host: "testhost",
+						Port: 80,
+					},
+				}
+
+				dns.EXPECT().Resolve("testhost").Return([]net.IP{net.ParseIP(testHostIP)}, nil).Once()
+			})
```

Comment:
Could we maybe add more tests to illustrate the statements we have in the code:
* LEN oldIPS <> LEN newIPs
* ip does not exists in new ip (maybe this test?)
* no changes in IPs